### PR TITLE
LSR-907 UserPortraitScreenlet error uploading big size images

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

File: ios/Framework/Core/Auth/UserPortraitScreenlet/UploadUserPortraitLiferayConnector.swift [hunk 1]
```diff
@@ -64,14 +64,16 @@ public class UploadUserPortraitLiferayConnector: ServerConnector {
...
 		}
 
-		if let imageBytes = UIImageJPEGRepresentation(src, CGFloat(factor)) {
+		let imageReduced = src.resizeImage(toWidth: Int(src.size.width * CGFloat(factor)))
+
+		if let imageBytes = UIImageJPEGRepresentation(imageReduced, 1) {
 			return (imageBytes.length < maxSize)
 				? imageBytes
-				: reduceImage(src, factor: factor - 0.05)
+				: reduceImage(src, factor: factor - 0.1)
```
Comment (jmnavarro) [inline ios/Framework/Core/Auth/UserPortraitScreenlet/UploadUserPortraitLiferayConnector.swift]: We need a hard limit. Images smaller than 100x100 won't be uploaded

Comment (victorg1991) [inline ios/Framework/Core/Auth/UserPortraitScreenlet/UploadUserPortraitLiferayConnector.swift]: Check it out :)  266be3a

File: ios/Framework/Core/Extensions/UIImage+Screens.swift [hunk 2]
```diff
@@ -16,22 +16,26 @@ import UIKit
 extension UIImage {
 
 	public func resizeImage(toWidth width: Int, completion: (UIImage) -> Void) {
+		dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0)) {
```
Comment (jmnavarro) [inline ios/Framework/Core/Extensions/UIImage+Screens.swift]: You can use our methods `dispatch_async` and `dispatch_sync`

Comment (victorg1991) [inline ios/Framework/Core/Extensions/UIImage+Screens.swift]: 5104da6 👍

**Comment by jmnavarro** [comment]
👍

**Pull request merged**